vector/src/main/res/layout/fragment_ftue_auth_use_case.xml

```diff
@@ -125,7 +123,6 @@
             android:padding="16dp"
             android:text="@string/ftue_auth_use_case_option_three"
             android:textColor="?vctr_content_primary"
-            app:drawableStartCompat="@drawable/ic_communities"
```

Comment:
Could be nice to keep those attributes but using the namespace `tools`, for a better later preview.

Reply:
I thought it didn't work and then realised I was only checking the blueprint view :sweat_smile: 

https://github.com/vector-im/element-android/pull/5025/commits/52dc5302c05a04c8d93d0d6c1669f73cb639f453

![2022-01-24T17:52:25,248487460+00:00](https://user-images.githubusercontent.com/1848238/150837591-495d6bba-303d-42d4-bf9b-ef8757962984.png)

Reply:
Why not using `ic_use_case_communities` and the 2 other drawables for the preview?

Reply:
<img width="295" alt="image" src="https://user-images.githubusercontent.com/3940906/150859001-9e298051-2f08-4e44-a61c-bd21cf6ef678.png">

Reply:
the `ic_use_case_*` drawables have intrinsic padding which can be a little misleading, I guess it depends if we would prefer the design preview to show relative sizing or the context of the image

no preference from my side!

Reply:
I see, OK